tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -22,8 +23,18 @@ class AuthorizationCheck(APIView):
     def get(self, request, *args, **kwargs):
         """Handle get request and verify user is authorized."""
         user = request.user
+        logger.info("AUTH_CHECK")
+        logger.info(str(user))
```

Comment:
```suggestion
```

Reply:
I'm good with deleting these. Also, can we alter the 3-line logger.info on the new L43/old L32 to be a one-liner to have consistent formatting?